sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -1693,6 +1735,11 @@ export interface ShareProperties {
     remainingRetentionDays?: number;
 }
 
+// @public
+export interface ShareProtocolSettings {
```

Comment:
Presumably other protocols are planned? Also, is this "selecting" `smb` in some sense, so we'd only expect the user to pass one key in this object?